packages/grafana-ui/src/components/Splitter/useSplitter.ts

```diff
@@ -21,6 +21,10 @@ export interface UseSplitterOptions {
    */
   onSizeChanged?: (flexSize: number, pixelSize: number) => void;
   onResizing?: (flexSize: number, pixelSize: number) => void;
+
+  // Size of the region left of the handle indicator that is responsive to dragging. At the same time acts as a margin
+  // pushing the left pane content left.
+  handleSize?: number;
```

Comment:
Personally, I would prefer if this was like `size="narrow"` or "sm" or something.

Be more opinionated and consistent.

Reply:
Yes I think that makes sense.